daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -166,6 +166,48 @@ def get_dpi_xy(self):
     return dpi
 
 
+@endpoint('razer.device.dpi', 'setDPIStages', in_sig='aq')
```

Comment:
I'm not sure if putting `set/getDPIStages` under `razer.device.dpi` is the best idea. Maybe it's better to add a a separate category such as `razer.device.dpi_stages`? Not all mice might implement dpi stages and it would be easier for clients to query dpi stages support before trying to use those methods.

Reply:
`razer.device.dpi` is fine